### PR TITLE
Add net.krafting.SemantiK.Lang.English

### PR DESCRIPTION
**Pull request by Krafting**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.

This is an Add-on for net.krafting.SemantiK, adding the English language pack for the model and wordlist used by the application. I used the addon system because language model files can be huge. (this one is +350MB).

This is my first Add-on, I hope I did it correctly, the PR for the required metadata/yml on the net.krafting.SemantiK is pending. https://github.com/flathub/net.krafting.SemantiK/pull/5
I will try building it locally after the new version is available on Flathub. In the meantime this is a draft, suggestions are still welcome.

I might add more language pack later on, so I will base myself on this one, tell me if you see anything I can improve.

Thank you for your time!

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. 

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: english.list
Comment (hfiguiere) [inline english.list:1]: why is this here? Flathub isn't to host ustream sources.

File: net.krafting.SemantiK.Lang.English.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - gzip -d GoogleNews-vectors-negative300-SLIM.bin.gz
+      - install -Dm644 GoogleNews-vectors-negative300-SLIM.bin /app/share/extra/English/model.bin
+      - install -Dm644 net.krafting.SemantiK.Lang.English.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo
+    sources:
+      - type: file
+        path: english.list
+      - type: file
+        path: net.krafting.SemantiK.Lang.English.metainfo.xml
+      - type: file
+        url: https://github.com/eyaler/word2vec-slim/raw/master/GoogleNews-vectors-negative300-SLIM.bin.gz
```
Comment (hfiguiere) [inline net.krafting.SemantiK.Lang.English.yml:23]: you should NEVER pull from a branch. You should pull a specific tag or commit.

File: english.list
Comment (Krafting) [reply english.list:1]: Moved it upstream. and used upstream git repo as source.

File: net.krafting.SemantiK.Lang.English.yml (hunk 1 above)
Comment (Krafting) [reply net.krafting.SemantiK.Lang.English.yml]: Fixed, used a commit and a separate step to download it.

**Comment by Krafting** [comment]
bot, build net.krafting.SemantiK.Lang.English

File: net.krafting.SemantiK.Lang.English.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+app-id: net.krafting.SemantiK.Lang.English
+build-extension: true
+separate-locales: false
+branch: '1.0.0'
```
Comment (bbhtt) [inline net.krafting.SemantiK.Lang.English.yml:4]: Do language packs needs to be branched ? This is used in case there are ABI breaks.

I think you can omit `version` in the extension point and omit this here

Comment (Krafting) [reply net.krafting.SemantiK.Lang.English.yml]: they would likely not cause ABI breaks, I was unsure if this was a required thing.

I remove it from here and a PR is waiting in the main App https://github.com/flathub/net.krafting.SemantiK/pull/6

**Comment by bbhtt** [comment]
bot, build net.krafting.SemantiK.Lang.English

**Comment by Krafting** [comment]
seems to be all good then
